src/shared/components/code-editor/CodeEditor.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { CSSProperties } from 'react';
+import { loader } from '@monaco-editor/react';
+import {
+  CodeEditor as PatternFlyCodeEditor,
+  Language,
+  EditorDidMount,
+} from '@patternfly/react-code-editor';
+import * as monacoConfig from 'monaco-editor/esm/vs/editor/editor.api';
+import { useShortcutPopover } from './hooks/useShortcutPopover';
+
+loader.config({ monaco: monacoConfig });
```

Comment:
this is to avoid using monaco from CDN - you can check it out in the lib's main README -> https://www.npmjs.com/package/@patternfly/react-code-editor

![Screenshot 2025-04-23 at 16 18 53](https://github.com/user-attachments/assets/c23bfea1-00b5-4d0d-85ec-e47e4d2ce9e4)